File: src/algorithms/remove-trailing-zeros.js
/**
 * Given a positive integer num represented as a string, returns the integer num
 * without trailing zeros as a string.
 * 
 * @param {string} num - A string representation of a positive number
 * @returns {string} - Returns num after removing trailing zeros
 */
function removeTrailingZeros (num) {
  for (let i = num.length - 1; i >= 0; i--) {
    if (num[i] !== '0') {
      return num;
    } else {
      num = num.slice(0, i);
    }
  }
}

File: src/algorithms/remove-trailing-zeros.py
def removeTrailingZeros(num):
    """Given a positive integer num represented as a string, return the integer 
    num without trailing zeros as a string.

    Args:
        num (str): A string of digits.

    Returns:
        str: Returns a modified string.
    """
    index = len(num) - 1
    while num[index] == "0":
        index -= 1
    return num[:index + 1]
